templates/discussions-dashboard.php

```diff
@@ -59,13 +59,18 @@
 }
 
 // If the referenced comment is not in the current batch of comments we need to re-add it.
-foreach ( $bulk_comments as $original_id => $_post_id ) {
-	if ( ! isset( $comments_by_post_id[ $_comment->comment_post_ID ] ) ) {
-		$linked_comment = $_comment->comment_content;
-		$parts          = wp_parse_url( $linked_comment );
-		$comment_id     = intval( str_replace( 'comment-', '', $parts['fragment'] ) );
-		if ( $comment_id ) {
-			$comments_by_post_id[ $_comment->comment_post_ID ][] = get_comment( $comment_id );
+foreach ( $bulk_comments as $original_id => $_comments ) {
```

Comment:
This is a fix for the error in line 76: because here we were adding a `null` entry to the `$comments_by_post_id` array and then sort that array, looking up that non-exist id in the `$latest_comment_date_by_post_id` array.